### DISCUSSION TITLE
移动超级SIM卡是用NFC-SIM卡技术？

### DISCUSSION THREAD

**Discussion by qhx826** [Ivysauro/CNRT#67] [Q&A]
NFC-SIM技术驱动的交通卡/ Transit Cards powered by NFC-SIM
大部分NFC-SIM卡已不再发行/ Most NFC-SIM cards are no longer issued

怪说不得，移动官方页面支持机型大多数新款机型都不支持了

**Comment by Ivysauro** [comment]
这一段几年前写的了😂

**Comment by chenw23** [comment]
已复核并纳入 PR #153。不能把“超级SIM”简单视为已停止发行的旧 NFC-SIM：中移动官方仍说明，超级SIM交通卡会将交通卡应用及个人化数据写入超级SIM卡，并通过支持NFC的手机刷卡乘坐公交、地铁。

因此仓库已删去绝对化的“多数 NFC-SIM 卡已不再发行”表述，改为现行超级SIM能力说明。

PR：https://github.com/Ivysauro/CNRT/pull/153
依据：https://dev.10086.cn/superSIM/portal-main/index
